packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -471,6 +471,7 @@ class PaginatedDataTableState extends State<PaginatedDataTable> {
                     sortColumnIndex: widget.sortColumnIndex,
                     sortAscending: widget.sortAscending,
                     onSelectAll: widget.onSelectAll,
+                    decoration: const BoxDecoration(),
```

Comment:
Can you add a comment for why you do this.

Reply:
Done!